webpack.wordpress.config.js

```diff
@@ -4,7 +4,7 @@ const webpack = require( 'webpack' );
 const LodashModuleReplacementPlugin = require( 'lodash-webpack-plugin' );
 
 module.exports = {
-	entry: './index.js',
+	entry: './targets/dist/index.js',
```

Comment:
The whole idea of separating core code in `src/` and target specific code in `targets/own-folder` is that targets only share code through `src/`; any other thing they need should be in `target/own-folder` directory but never depend on a different target.

I haven't looked at the further PR yet, so I'm unsure where this is leading, but perhaps in this step we still need the root `index.js` and refactor this later when other pieces are in place?

Reply:
> The whole idea of separating core code in src/ and target specific code in targets/own-folder is that targets only share code through src/; any other thing they need should be in target/own-folder directory but never depend on a different target.

I agree, but that is only partially correct because right now multiple targets share the same entry point `./index.js` which is not part of `src/`. Ideally and what I was trying to do is to have an entry point for each target instead of having a shared `./index.js`. I updated the `WordPress` target to use  `dist` target based on the assumption that the WordPress plugin will always use the same browser interface as the `dist` target. Which was also the case when sharing `./index.js` entry point. But for the sake of consistency I will add an `index.js` entry point to the WordPress target which will basically be identical with the entry for `dist` so that we don't make WordPress plugin target dependent on the `dist` target.

> I haven't looked at the further PR yet, so I'm unsure where this is leading, but perhaps in this step we still need the root index.js and refactor this later when other pieces are in place?

I'd rather have a better separation of code and targets from start.

Reply:
I don't disagree with the idea of moving the entry point to every target's directory instead of having a common one - either way, the separation between `src/` and `targets` is clear (and I actually hesitated between the two options). But I feel strongly about that separation (as it provides helpful abstractions for the library), so I would recommend the WordPress target doesn't depend on other target assets.

Reply:
> But I feel strongly about the separation between core and targets, so I would recommend the WordPress target doesn't depend on other target assets.

I agree, WordPress has its own entry point now.